resources/hpxml.rb

```diff
@@ -2055,22 +2089,38 @@ def self.add_layer_insulation(parent:,
       layer = XMLHelper.add_element(insulation, "Layer")
       XMLHelper.add_element(layer, "InstallationType", "cavity")
       XMLHelper.add_element(layer, "NominalRValue", Float(cavity_nominal_r_value)) unless cavity_nominal_r_value.nil?
+      layers[:exterior] = layer
     end
     unless continuous_nominal_r_value.nil?
       layer = XMLHelper.add_element(insulation, "Layer")
       XMLHelper.add_element(layer, "InstallationType", "continuous")
       XMLHelper.add_element(layer, "NominalRValue", Float(continuous_nominal_r_value)) unless continuous_nominal_r_value.nil?
+    else
+      unless ext_continuous_nominal_r_value.nil?
```

Comment:
Add ext/int continuous path for foundation wall to keep other function calls untouched, add layer elements as return values, to proceed with adding extension.